pkgs/applications/networking/cluster/kuma/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, coredns
+, installShellFiles
+, isFull ? false
+, enableGateway ? false
+, pname ? "kuma"
+, components ? lib.optionals isFull [
+    "kumactl"
+    "kuma-cp"
+    "kuma-prometheus-sd"
+    "kuma-dp"
+  ]
+, version ? "1.4.0"
+, tags ? lib.optionals enableGateway ["gateway"]
+, rev ? version
+, sha256 ? "0agib3w5s270n5rhg54m3p4f21s5fhdrhaks374j7l21njs7xhqb"
+, vendorSha256 ? "1fc5psvbd9bpc6c3y2cpx5dx8cgr2fcp7nln3kwfgbryahq2y8wl"
```

Comment:
We are not overriding any of those and they can always be overriden by overrideAttr's. Lets remove them.

Reply:
Many thank's for review. makes sense. I changed it to field inside set